users/serializers.py

```diff
@@ -27,4 +28,19 @@ def update(self, instance, validated_data):
             else:
                 setattr(instance, attr, value)
         instance.save()
-        return instance
\ No newline at end of file
+        return instance
```

Comment:
You should not allow password to be updated here. You can use super() of the update and create methods.